DotNetStandardApi/Session.cs

```diff
@@ -844,6 +846,23 @@ public async Task<Entities.ZoneSummary> SwitchHeatingOff(int homeId, int zoneId,
             return await SetTemperature(homeId, zoneId, null, null, Enums.DeviceTypes.Heating, durationMode, timer);
         }
 
+        /// <summary>
+        /// Sets the Home Presence mode manually, regardless of current geofence status of home devices.
+        /// </summary>
+        /// <param name="homeId">Id of the home to set the home presence for.</param>
+        /// <param name="presence">Presence to set for the home.</param>
+        /// <returns>Boolean indicating if the request was successful</returns>
+        /// <exception cref="ArgumentOutOfRangeException">Thrown when an invalid enum value is used.</exception>
+        public Task<bool> SetHomePresence(int homeId, HomePresence presence)
+        {
+            EnsureAuthenticatedSession();
+            EnumValidation.EnsureEnumWithinRange(presence);
```

Comment:
In what situation could this throw an exception? If you pass in the enum, how can it receive a value that's not in that enum?

Reply:
```csharp
await session.SetHomePresence(homeId, (HomePresence)7);
```

Just ensuring that the `ToString()` call does not fail and invalid values are not sent to Tado 😊.

Reply:
Nice one. Never have known this weird trick to mess with enums :)